lib/prometheus/business_collector.rb

```diff
@@ -64,6 +70,28 @@ def collect
       Host.with_policies_or_test_results
           .where(account: client_accounts.select(:account_number)).count
     )
+
+    Host.with_policies_or_test_results.select(
+      "COUNT(id), concat(
+        #{Host::OS_MAJOR_VERSION},
+        '.',
+        #{Host::OS_MINOR_VERSION}
+      ) as version"
+    ).group('version').map(&:attributes).each do |item|
+      @total_systems_by_os.observe(item['count'], version: item['version'])
+    end
+
+    Host.with_policies_or_test_results.where(
+      account: client_accounts.select(:account_number)
+    ).select(
+      "COUNT(id), concat(
+        #{Host::OS_MAJOR_VERSION},
+        '.',
+        #{Host::OS_MINOR_VERSION}
+      ) as version"
+    ).group('version').map(&:attributes).each do |item|
+      @client_systems_by_os.observe(item['count'], version: item['version'])
+    end
```

Comment:
Be sure to add tests to business_collector_test.rb :)

Reply:
Fixed